index.ts

```diff
@@ -0,0 +1,217 @@
+import * as adapter from "@api3/airnode-adapter";
+import * as node from "@api3/airnode-node";
+import { AirnodeRrpFactory } from "@api3/airnode-protocol";
+import * as dotenv from "dotenv";
+import * as ethers from "ethers";
+import * as fs from "fs";
+import { each, isEmpty } from "lodash";
+import * as path from "path";
+import { safeDecode } from "./node/abi-encoding";
+//TODO: remove and use @api3/airnode-node import
+import { buildEVMProvider } from "./node/evm-provider";
+//TODO: remove and use @api3/airnode-node import
+import { removeKey, removeKeys } from "./node/object-utils";
+//TODO: remove and use @api3/airnode-node import
+import { getReservedParameters, RESERVED_PARAMETERS } from "./node/parameters";
+//TODO: remove and use @api3/airnode-node import
+import { deriveSponsorWallet } from "./node/wallet";
```

Comment:
Are these even exported?

Reply:
They are not and that's why I temporarily copied the code over to airkeeper repo. I made the assumption that they are going to be exported in the airnode repo tho.

Reply:
Okay, I guess that will be in 0.3.0?

Reply:
Yes, working on this today